main.rb

```diff
@@ -1,5 +1,74 @@
 require_relative 'app'
 
+class Functions
+  def create_person(app)
+    puts 'Do you want to create a student(1) or a teacher(2)? [Input the number]:'
+    option = gets.chomp
+    parent_permission = true
+    puts 'age:'
+    age = gets.chomp.to_i
+    puts 'name:'
+    name = gets.chomp
+
+    case option.to_i
+    when 1
+      puts 'Has parent permission?[Y/N]'
+      val = gets.chomp
+      parent_permission = false if %w[N n NO no No].include?(val)
+      app.create_student(age, name, parent_permission)
+    when 2
+      puts 'specialization:'
+      specialization = gets.chomp
+      app.create_teacher(age, specialization, name)
+    else
+      puts 'Invalid input :('
+    end
+  end
+
+  def handle_option(option, app)
+    case option
+    when 3
+      create_person(app)
+    when 4
+      create_book(app)
+    when 5
+      create_rental(app)
+    when 6
+      puts 'Enter person ID:'
+      id = gets.chomp.to_i
+      app.list_person_rentals(id)
+    when 7
+      puts 'Thanks :)..'
+      return false
+    else
+      puts 'Invalid number :('
+    end
+    true
+  end
+
+  def create_book(app)
+    puts 'Title:-'
+    title = gets.chomp
+    puts 'Author:-'
+    author = gets.chomp
+    app.create_book(title, author)
```

Comment:
- [ ] There is no method called `create_book` in the app.rb file so I am unable to create a book. Kindly fix that. The suggestion from the previous reviewer was that you get rid of the function file but move the methods to the main.rb file. Kindly ensure your app works completely before re-submitting to avoid wasting review counts.

  ![fjfif](https://github.com/jeanbuhendwa/school-library/assets/54848363/a6ea859f-fa6e-4322-bfef-0f47f7abd277)